.circleci/config.in.yml

```diff
@@ -63,7 +68,8 @@ jobs:
     resource_class: 2xlarge+
     steps:
       - checkout
-      - run: test_community_repos/run_all.sh
+      - run: packaging/build_wheel.sh
```

Comment:
You can't do this. `build_wheel.sh` sets a number of environment variables locally, but after the shell script exits you lose all of that state. If you can't put the execution in the script itself, you'll have to propagate the env vars back out to CircleCI to be persisted for further steps. https://circleci.com/docs/2.0/env-vars/#using-bash_env-to-set-environment-variables